client/src/pages/Notifications/Notifications.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useEffect } from 'react';
+import Paper from '@material-ui/core/Paper';
+import Pagination from '@material-ui/lab/Pagination';
+import Typography from '@material-ui/core/Typography';
+import Box from '@material-ui/core/Box';
+import useStyles from './NotificationStyles/Notifications';
+import NotificationMessages from '../../components/Notification/NotificationMessages';
+import { getNotifications, updReadNotifications, getCount } from '../../helpers/APICalls/notifications';
+import { Notification } from '../../interface/Notifications';
+
+export default function NotificationPopover(): JSX.Element {
+  const classes = useStyles();
+  const returnNotificationType = (res: any): Notification[] => res.notifications;
+  const messages: Notification[] = [];
+  const [messagesToShow, setmessagesToShow] = React.useState(messages);
+  const [page, setPage] = React.useState(1);
+  const [count, setCount] = React.useState(1);
+
+  const handleChange = (event: React.ChangeEvent<unknown>, value: number) => {
+    setPage(value);
+  };
+
+  useEffect(() => {
+    const fetchNotifications = async () => {
+      const notificationRes = await getNotifications(page, 20, null);
+      const messagesCopy = returnNotificationType(notificationRes);
+      if (messagesCopy) setmessagesToShow(messagesCopy);
+
+      const unreadMessages = messagesCopy.filter((notifs) => !notifs.read).map((notifs) => notifs._id);
+      if (unreadMessages.length <= 0) return;
+      await updReadNotifications(unreadMessages);
+    };
+    fetchNotifications();
+  }, [page]);
+
+  useEffect(() => {
+    const fetchCount = async () => {
+      const success = await getCount(null);
+      if (success) {
+        const pageLimit = 20;
+        const countCalc = Math.ceil(success['count'] / pageLimit);
+        setCount(countCalc);
+      }
+    };
+    fetchCount();
+  }, []);
+
+  return (
+    <Box className={`${classes.root} ${classes.pageWrap}`}>
+      <Paper className={classes.paper}>
+        {messagesToShow.length < 1 ? (
```

Comment:
This can just be `messagesToShow.length` :)

Ditto throughout!

Reply:
done